migrations/sqls/20190910195603-services-update-09102019-down.sql

```diff
@@ -0,0 +1,15 @@
+UPDATE services
+SET serviceName = "Workers' Rights and Safety"
+WHERE serviceName = "Worker's Rights"
+;
+
+DELETE FROM services
```

Comment:
This down migration will fail because of the foreign keys on the services table from `intakeServices` and `clientInteractions`. We should maybe just disable foreign key checks and re-enable at the end of this migration? Or do something with the rows in the intakeServices and clientInteractions tables (maybe delete them)